### ISSUE TITLE
Landmarks: accessible names

### ISSUE THREAD

**Issue by shawnthompson** [wet-boew/wet-boew#9072]
All landmarks should have accessible names to them. Just having a heading in them is not enough.

Multiple instances of the same type of landmark SHOULD be distinguishable by different programmatically determinable labels (`aria-label` or `aria-labelledby`).

**Comment by duboisp** [comment]
Ok, do we need to add that to all sectioning content?

When or what software this cause an issue? what test has been run?

The current markup seem conform to WhatWG spec: https://html.spec.whatwg.org/multipage/sections.html#the-nav-element
